### PR TITLE
Parametrise configurations

### PR DESCRIPTION
**Pull request by skovbakke**
Added parameters to the Makefile allowing config and requirement files to be specified on the command-line. This is usefull when environment variables are stored in a separate location. 

Also did some small renaming & refactoring. There was an issue where the config script would fail when the number of workers was zero and the `build/cluster/worker` directory did not exist.

### CODE REVIEW

File: scripts/pack_helm_charts.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 DIR=$1
 mkdir $DIR -p
-cat helm_requirements.txt | grep -w ".*http.*"   | while read p; do
+cat helm_requirements | grep -w ".*http.*"   | while read p; do
```
Comment (jacobcrawford) [inline scripts/pack_helm_charts.sh:4]: Should this file not be referenced by the HELM_REQUIREMENTS variable?

File: scripts/pack_helm_charts.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ cat helm_requirements.txt | grep -w ".*http.*"   | while read p; do
 done
 
 numberOfFiles=$(ls $DIR | wc -l)
-linesInFile=$(grep -w ".*http.*" -c helm_requirements.txt)
+linesInFile=$(grep -w ".*http.*" -c helm_requirements)
```
Comment (jacobcrawford) [inline scripts/pack_helm_charts.sh:15]: same

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Should the image_requirements and helm_requirements files not be referenced by the environment variables?
Otherwise looks good.

File: scripts/pack_helm_charts.sh (hunk 2 above)
Comment (skovbakke) [reply scripts/pack_helm_charts.sh]: yes

File: scripts/pack_helm_charts.sh (hunk 1 above)
Comment (skovbakke) [reply scripts/pack_helm_charts.sh]: yes

**Review by jacobcrawford — APPROVED** [review]
looks good.

**Pull request merged**